test.js

```diff
@@ -14,6 +14,10 @@ function commit (msg) {
   shell.exec('git commit --allow-empty -m"' + msg + '"')
 }
 
+function execCli (argString) {
+  return shell.exec('node ' + cliPath + ' ' + argString)
```

Comment:
Great catch! To allow falsy values without allowing undefined or null I changed it to:
```js
return shell.exec('node ' + cliPath + (argString != null ? ' ' + argString : ''))
```